docs/sections/user_guide/installation.rst

```diff
@@ -1,10 +1,9 @@
-.. include:: links.rst
-
-************
 Installation
-************
+============
+
+.. include:: links.rst
 
--  If you are a **developer**, please visit the `Developer Setup <../contributor_guide/developer_setup.html>`_ section located in the `Contributor Guide <../contributor_guide/index.html>`_.
+-  If you are a **developer**, please visit the :doc:`Developer Setup <../contributor_guide/developer_setup>` section located in the :doc:`Contributor Guide <../contributor_guide/index>`.
```

Comment:
I found this more RST-idiomatic way to link to pages in the same document tree, without mentioning or assuming HTML filenames.